src/ui/components/Header/Header.js

```diff
@@ -25,9 +27,37 @@ const GET_RECORDING_TITLE = gql`
   }
 `;
 
+function Avatars({ recordingId }) {
+  const { data, loading } = useGetActiveSessions(recordingId);
+
+  if (loading) {
+    return null;
+  }
+
+  const uniqueUsers = groupBy(data.sessions, session => session.user?.auth_id);
+  const displayedSessions = Object.keys(uniqueUsers).reduce((acc, key) => {
+    if (key == "undefined") {
+      return [...acc, ...uniqueUsers[key]];
+    }
+
+    return [...acc, uniqueUsers[key][0]];
+  }, []);
+  const displayedUsers = displayedSessions.map(session => session.user);
```

Comment:
this feels overly complicated...

why can't I do `uniqBy(sessions.map(session => session.user), user => user.id)`

Reply:
we'll also need a nice way to remove the logged in user

Reply:
Yes - this was done in a roundabout way because there's a way to differentiate session for users, but no way to differentiate sessions for guests.

There's three cases to consider:
1) Recording has me, you and two guests looking at it. 
- A: [Jaril, Jason, Guest1, Guest2]
- B: [Jason, Guest1, Guest2]
2) Recording has me, you (but opened in two tabs), and two guests are looking at it. 
- A: [Jaril, Jason, Jason, Guest1, Guest2]
- B: [Jason, Guest1, Guest2]
3) Recording has me, you, and one guest (but opened in two tabs) looking at it 
- A: [Jaril, Jason, Guest1, Guest1]
- B: [Jason, Guest1]

As a general rule, it's obvious that we shouldn't show two avatars for one user just because they have multiple sessions (1B, 2B). Problem is when it comes to guests – we can't tell the difference between 1 guest looking at 2 tabs, or 2 guests looking at 1 tab each. Both of those sessions come back as `user: null`.

I thought that we should just treat each guest session as a separate avatar (3B). Should we do differently?